src/plugins/controls/public/control_group/editor/edit_control_group.tsx

```diff
@@ -1,88 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0 and the Server Side Public License, v 1; you may not use this file except
- * in compliance with, at your election, the Elastic License 2.0 or the Server
- * Side Public License, v 1.
- */
-
-import React from 'react';
-import { EuiContextMenuItem } from '@elastic/eui';
-
-import { toMountPoint } from '@kbn/kibana-react-plugin/public';
-import { OverlayRef } from '@kbn/core/public';
-import { ControlGroupStrings } from '../control_group_strings';
-import { ControlGroupEditor } from './control_group_editor';
-import { pluginServices } from '../../services';
-import { ControlGroupContainer } from '..';
-import { setFlyoutRef } from '../embeddable/control_group_container';
-
-export interface EditControlGroupButtonProps {
-  controlGroupContainer: ControlGroupContainer;
-  closePopover: () => void;
-}
-
-export const EditControlGroup = ({
-  controlGroupContainer,
-  closePopover,
-}: EditControlGroupButtonProps) => {
```

Comment:
Good call to switch this over. Returning components from our APIs is not a very good pattern!